charts/bindplane/templates/bindplane.yaml

```diff
@@ -48,6 +48,13 @@ spec:
             {{- if .Values.config.license }}
             - name: BINDPLANE_LICENSE
               value: {{ .Values.config.license }}
+            {{- else if .Values.config.licenseUseSecret }}
```

Comment:
If config.license is not set, and licenseUseSecret is true, mount license from the secret.